aiida/orm/data/code.py

```diff
@@ -152,33 +145,27 @@ def get_code_helper(cls, label, machinename=None):
         :raise MultipleObjectsError: if the string cannot identify uniquely
             a code
         """
-        from aiida.common.exceptions import (NotExistent, MultipleObjectsError,
-                                             InputValidationError)
+        from aiida.common.exceptions import (NotExistent, MultipleObjectsError, InputValidationError)
         from aiida.orm.querybuilder import QueryBuilder
         from aiida.orm.computers import Computer
 
         qb = QueryBuilder()
-        qb.append(cls, filters={'label': {'==': label}},
-                  project=['*'], tag='code')
+        qb.append(cls, filters={'label': {'==': label}}, project=['*'], tag='code')
         if machinename:
-            qb.append(Computer, filters={'name': {'==': machinename}},
-                  computer_of='code')
+            qb.append(Computer, filters={'name': {'==': machinename}}, computer_of='code')
 
         if qb.count() == 0:
-            raise NotExistent("'{}' is not a valid code "
-                              "name.".format(label))
+            raise NotExistent("'{}' is not a valid code " "name.".format(label))
```

Comment:
There are a few of `" "` around, I guess after yapf, that could be fixed, maybe in a different PR.

Reply:
Yes, I will make a note, we can do one great sweep at the end of this project